storage/block/how-to/create-a-volume.mdx

```diff
@@ -11,27 +11,23 @@ dates:
 ---
 
 <Message type="requirement">
-
-- You have an account and are logged into the [Scaleway Console](https://console.scaleway.com/) 
-
+  - You have an account and are logged into the [Scaleway Console](https://console.scaleway.com/) 
 </Message>
 
 
 1. Click **Instances** in the **Compute** section of the [Scaleway Console](https://console.scaleway.com) side menu. The Instances section displays.  
 2. Click **Volumes** to display a list of all your current volumes. 
-  <Lightbox src="scaleway-volumes-list.webp" alt="" />
+    <Lightbox src="scaleway-volumes-list.webp" alt="" />
 3. Click **Create a Volume** to create a volume that can be attached to any of your Instances.
-  <Lightbox src="scaleway-create-volume.webp" alt="" />
+    <Lightbox src="scaleway-create-volume.webp" alt="" />
 4. Choose a region for the volume.
   <Message type="important">
+      The volume must be in the same region as the Instance you want to connect it to.
+    </Message>
 
-  The volume must be in the same region as the Instance you want to connect it to.
-
-  </Message>
-
-  <Lightbox src="scaleway-choose-region.webp" alt="" />
+    <Lightbox src="scaleway-choose-region.webp" alt="" />
 5. Select Block Storage when configuring the volume. Enter the volume name and select a size.
-  <Lightbox src="scaleway-configure-block-volume.webp" alt="" />
+    <Lightbox src="scaleway-configure-block-volume.webp" alt="" />
 6. Click **Create a new volume** to confirm the creation of the volume.
 
 <Navigation title="See Also">
```

Comment:
Navigation next button should be in right position.

Here it is in left at local build:
![Capture d’écran 2022-10-19 à 10 20 47 AM](https://user-images.githubusercontent.com/341803/196637146-5e8bda6f-d163-47f0-99e1-54ca87eff1a8.png)

But in right at prod:

![Capture d’écran 2022-10-19 à 10 20 52 AM](https://user-images.githubusercontent.com/341803/196637268-6835fdec-d4d0-4dac-a539-de4943b7bb31.png)

Reply:
C'est ok chez moi en build local ici